Formula/cbmc.rb

```diff
@@ -0,0 +1,39 @@
+class Cbmc < Formula
+  desc "CBMC: The C Bounded Model Checker"
+  homepage "https://www.cprover.org/cbmc/"
+  url "https://github.com/diffblue/cbmc/archive/cbmc-5.12-d8598f8.tar.gz"
```

Comment:
Is the hash not part of the version?

Reply:
I was using the wrong package for cbmc 5.12, I have fixed this in the formula.